### ISSUE TITLE
[FEATURE REQ] When scrolled to bottom of page, lock it until you scroll up manually.

### ISSUE THREAD

**Issue by kman3107** [HaveAGitGat/Tdarr#1024]
**Is your feature request related to a problem? Please describe.**
It is very frustrating when the page jumps up and down when you try to click on requeue or bump.
I have many times clicked the wrong (button) file because of this.

**Describe the solution you'd like**
I wish that when I scroll all the way to the bottom the page do not contract and stretch vertically at the bottom.
This way I can safely click buttons and not worry that I'll requeue the entire transcode tree.

**Describe alternatives you've considered**
Sorry, got nothing here.

**Additional context**

![Tdarr_Jump](https://github.com/HaveAGitGat/Tdarr/assets/10728652/aa95424f-fc18-4de4-9a03-5d48599c461c)

**Comment by HaveAGitGat** [comment]
Hi, I assume you mean this is due to the worker section changing size due to the filename or status of the worker:

![image](https://github.com/HaveAGitGat/Tdarr/assets/43864057/6f128386-2f60-4ec7-842e-aee336032a09)

So with long filenames or status it will linebreak, increasing the section size. I can change this so it does not line break:
![image](https://github.com/HaveAGitGat/Tdarr/assets/43864057/280323ba-586a-4e4a-8c89-a87563d9f751)

But then no doubt will get complaints from people saying they can't see the full filename so I don't think that's a good idea. If you're doing lots of file actions then it's recommended to pause processing as it could also be that right before you click an item in the table, the table updates and a new item appears in the row for the item that you were previously going to click. But yes open to suggestions, each section most likely needs it's own page so the sections don't affect eachother.

It should not be possible to accidentally requeue all transcode items as there's a confirmation prompt if the Requeue all button is pressed:

![image](https://github.com/HaveAGitGat/Tdarr/assets/43864057/76ca3093-37a8-484a-adc1-4bb30ec14c1f)
 
Please let me know if that does not show for you.

**Comment by kman3107** [comment]
Taking a look at it, it does seem to be the linebreak, and I agree that it's not the best idea to cut the name 😄 
I unfortunately do not have any suggestion on how to handle this problem. This is just a feature request, so I don't expect high priority on this unless someone can actually give a full-fledged idea on how to deal with this. I just thought it made sense to make the request so it is out there.

As for the confirmation pop-up, it did indeed show.
It frustrating to me because my hands and fingers are not steady, and I have on **many** occasions clicked something I was trying **not** to click. No, most of the time my issue in Tdarr has been clicking one of the other single files.

I just started my server 3 months ago and found tdarr less than 2 months ago. I started my collection of TV and movies over 20 years ago so I have a lot to get through, plus I upgrade the quality of files all the time. So I am still in the catching up phase.

Anyway, thank you for the amazing software! You have lessened the stress of me and my storage immensely!
